doc/lua_api.md

```diff
@@ -995,6 +995,60 @@ minetest.register_node("default:dirt_with_grass", {
 })
 ```
 
+Variants
+--------
+
+Items can have "variants", which are numbered states that can determine certain
+properties. The number of variants is specified with the item definition field
+`variant_count`. Each variant is numbered from 0 to `variant_count` - 1. Every
+item has at least one variant (numbered 0). Nodes can specify a part of param2
+to read as the variant number by setting the `param2_variant` field to a
+`BitField`.
+
+The currently supported variant properties are `tiles`, `overlay_tiles`, and
+`special_tiles`. The `variants` table in the item definition is a mapping from
```

Comment:
Can this be an itemized list?
That would be easier to understand.

Reply:
Should be resolved.